### PR TITLE
Uses title HTML tag whenever is possible to update the page title. Closes #185

### LINKED ISSUE #185: If the HTML title contains character entities, the title bar / tab does not render correctly
### Expected behaviour

`<title>left &amp; right</title>` will render as "left & right" in the title bar, no matter how the page is loaded.

### Actual behaviour

`<title>left &amp; right</title>` will render as "left & right" in the title bar, only if the page is loaded directly. It will render as "left &amp;amp; right" if the page is loaded through Senna.js

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by SpencerWoo** [comment]
@fernandosouza I don't believe this passes the tests of 
https://github.com/brunobasto/senna.js/pull/8/commits/730432392b89f38e4f210bce61f89dd1335065b5

or a test of resolveTitleFromVirtualDocument in HtmlScreen.js such as 
`it('should handle html entities in setting the title', (done) => {
		var screen = new HtmlScreen();
		screen.load('/url').then(() => {
			assert.strictEqual('< > &', screen.getTitle());
			done();
		});
		this.requests[0].respond(200, null, '<title>< > &</title>');
	});`

File: src/app/App.js [hunk 1]
```diff
@@ -1106,7 +1106,14 @@ class App extends EventEmitter {
 		} else {
 			globals.window.history.pushState(state, title, path);
 		}
-		globals.document.title = title;
+
+		let titleNode = globals.document.querySelector('head title');
```
Comment (eduardolundgren) [inline src/app/App.js]: Only `document.querySelector('title');` is enough here

**Comment by fernandosouza** [comment]
You are right @SpencerWoo. I'm going to fix the implementation. Thank you.

**Comment by fernandosouza** [comment]
Pull updated. @eduardolundgren, please don't merge yet. I'm going to send another pull making Travis run the tests over Saucelabs for PRs.

**Pull request merged**